src/main/webapp/dashboardPresidente.jsp

```diff
@@ -82,7 +82,7 @@
         <!-- ============================================================== -->
         <!-- Topbar header - style you can find in pages.scss -->
         <!-- ============================================================== -->
-       	<%@ include file="header.jsp" %>
+       	<%//@ include file="header.jsp" %>

```

Comment:
Come mai hai rimosso sia l'header che il menu?

Reply:
Non avviava la dashboard in locale e poi li ho dimenticati, ora li rimetto